### DISCUSSION TITLE
Huisnummer

### DISCUSSION THREAD

**Discussion by Levisz92** [woocommerce/woocommerce#46320] [Checkout Flow]
ik ken nergens een huisnummer invoeren bij afrekenen ik proberen de vezendinfo blok aan te passen maar alles is gelocked

**Comment by nielslange** [comment]
Thanks for reaching out, @Levisz92. I moved this case to https://github.com/woocommerce/woocommerce/issues/46324, as it's more an issue than a discussion item.
